exasol_advanced_analytics_framework/event_context/udf_event_context.py

```diff
@@ -1,10 +1,18 @@
 from collections import OrderedDict
-from typing import Union,  Mapping
-from exasol_advanced_analytics_framework.context_wrapper.context_wrapper_base \
-    import ContextWrapperBase
+from typing import Union, Mapping, List
 
+from exasol_data_science_utils_python.preprocessing.sql.schema.column import \
+    Column
+from exasol_data_science_utils_python.preprocessing.sql.schema.column_name import \
+    ColumnName
+from exasol_data_science_utils_python.preprocessing.sql.schema.column_type import \
+    ColumnType
 
-class UDFContextWrapper(ContextWrapperBase):
+from exasol_advanced_analytics_framework.event_context.event_context_base \
+    import EventContextBase
+
+
+class UDFEventContext(EventContextBase):
     def __init__(self, ctx, column_mapping: Mapping[str, str],
```

Comment:
We need to inject the exa object via the constructor for getting the column types